src/Quotebot/Services/CommandsHandlerService.cs

```diff
@@ -1,12 +1,10 @@
-﻿using Discord;
-using Discord.Commands;
-using Discord.WebSocket;
+﻿using Discord.Commands;
 using Microsoft.Extensions.DependencyInjection;
 using System.Reflection;
 
 namespace Quotebot.Services
 {
-    internal class CommandsHandlerService
+    public class CommandsHandlerService
```

Comment:
why did this have to be public?

Reply:
DI gets upset if a public class tries to inject an internal class, so we can go with internal, but we'd have to make sure every class that consumes this is internal

Reply:
Specifically, Bot.cs was upset that CommandsHanderService is internal because Bot is public